STEERING-COMMITTEE.md

```diff
@@ -265,6 +274,24 @@ new person to the end user seat.
 
 The person filling the end user seat will take office in February.
 
+## Selected Seats
```

Comment:
This section is redundant as the definition of "selected seats" is already in the "Composition" section. I'd suggest either weaving in this information here to the definition of _selected seats_ above or having third-level sections for "### Contributing Seats", "### End User Seat" and "### Selected Seats" in the section "## Composition" above.